engine/collection/compliance/core.go

```diff
@@ -139,7 +139,7 @@ func (c *Core) OnBlockProposal(proposal flow.Slashable[*messages.ClusterBlockPro
 		log = log.With().Strs("tx_ids",
 			flow.IdentifierList(block.Message.Payload.Collection.Light().Transactions).Strings()).Logger()
 	}
-	log.Info().Msg("block proposal received")
+	log.Debug().Msg("block proposal received")
```

Comment:
can we leave this one? it marks start of processing a block proposal in compliance engine

Reply:
I reverted this to be info-level again (https://github.com/onflow/flow-go/pull/7174/commits/cd74b5dd772b92c22bdb01b600cedffd4fdfa3f8). I also made the log between hotstuff processing and protocol state processing debug-level (https://github.com/onflow/flow-go/pull/7174/commits/a2faab8bf394c9c6839f9f631dbe7b9ebc7e4e5e)